noetic/distribution.yaml

```diff
@@ -5969,8 +5969,76 @@ repositories:
       test_pull_requests: true
       type: git
       url: https://github.com/ros-visualization/python_qt_binding.git
-      version: melodic-devel
+      version: noetic-devel
```

Comment:
This is an unrelated change that is pointing to a non-extant branch. Please don't make unrelated changes.

Reply:
yes it is what I want